### PR TITLE
Update 892e11f7-ef46-47ed-aba2-7efc2ea83c60.md

### PR DESCRIPTION
**Pull request by helen-laktionova**
Please double check with Sergej Lapin

### CODE REVIEW

**Review by lapin-sergej — APPROVED** [review]

File: 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ Clickjacking is a possibility to tweak the UI or craft a malicious webpage, embe
 
 ### Obsolete or outdated dependencies
 Keep your dependencies up-to-date.
+Make sure that:
+*   You have all the security updates from Spryker modules.
```
Comment (andriitserkovnyi) [inline 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md:83]: What do you mean by "all" security updates from Spryker modules? Should it be "All your modules got all the security updates"?

File: 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md [hunk 2]
```diff
@@ -79,6 +79,9 @@ Clickjacking is a possibility to tweak the UI or craft a malicious webpage, embe
...
+*   You have all the security updates from other modules.
```
Comment (andriitserkovnyi) [inline 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md:84]: Could you give an example of a non-Spryker module?

Comment (helen-laktionova) [reply 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md:84]: f.e Symfony

File: 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md (hunk 1 above)
Comment (helen-laktionova) [reply 202108.0/892e11f7-ef46-47ed-aba2-7efc2ea83c60.md:83]: yep, 
but for Spryker modules, you will need to check Spryker security updates, security Emails if updates were not published yet. 
For others - you can just use some 3rd party system to identify missing security updates. 
Thats why I want to have them separated

**Pull request merged**